symplyphysics/core/symbols/quantities.py

```diff
@@ -48,7 +48,7 @@ def _eval_is_positive(self) -> bool:
         return self.scale_factor >= 0
 
     def _eval_Abs(self) -> Self:
-        return self if self.scale_factor >= 0 else (-1 * self)
+        return Quantity(Abs(self.scale_factor), dimension=self.dimension)
```

Comment:
Mypy complains about this.
Use `return self.__class__(Abs(self.scale_factor), dimension=self.dimension)`